src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/config/PrisonConfiguration.kt

```diff
@@ -0,0 +1,21 @@
+package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.config
+
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import java.time.LocalTime
+
+@Configuration
+class PrisonRegimeConfiguration {
```

Comment:
Not really sure on this, it is just to get something in place.  This will likely change/evolve.